### PR TITLE
Scraping Tool Kit fixes

### PR DESCRIPTION
**Pull request by russell-d-e**
Made updates to use CONSTANTS.py to handle situations where the ingestion nodegroup name has invalid characters  for python function calls. ReadNodegroups.py was updated to correctly get the nodegroup name regardless of os.

### CODE REVIEW

**Review by AbhaMoitra — APPROVED** [review]
The changes look fine to me, and we should test them out in the dev branch.

File: ScrapingToolKit/README.md [hunk 1]
```diff
@@ -7,6 +7,9 @@ STK is installed by downloading the files to your local machine.  This can eithe
 
 Dependencies:
 `pip install lxml`
```
Comment (glguy) [inline ScrapingToolKit/README.md:9]: for these we could tell people to run:

```
pip install -r requirements.txt
```

Comment (russell-d-e) [reply ScrapingToolKit/README.md:10]: updated readme to tell people to install dependencies via requirements.txt

**Comment by tuxji** [comment]
Is this pull request still relevant?  It should either be merged or closed, not left to linger for a long time.

**Pull request merged**